redaxo/src/core/lib/login/backend_password_policy.php

```diff
@@ -9,18 +9,134 @@ class rex_backend_password_policy extends rex_password_policy
 {
     use rex_factory_trait;
 
-    public function __construct(array $options)
+    /** @var int|null */
+    private $noReuseOfLast;
+    /** @var DateInterval|null */
+    private $noReuseWithin;
+    /** @var DateInterval|null */
+    private $forceRenewAfter;
+    /** @var DateInterval|null */
+    private $blockAccountAfter;
+
+    public function __construct(?array $options = null)
```

Comment:
soo irgendwie müsste das gehen, oder?

```suggestion
    /**
     * @param $options array{?no_reuse_of_last: ?int, ?no_reuse_within: ?DateInterval, ?force_renew_after: ?DateInterval, ?block_account_after: ?DateInterval}
     */
    public function __construct(?array $options = null)
```

Reply:
eigentlich können wir den parameter auch ganz entfernen. wenn jemand die rules trotzdem explizit übergibt, gibt es ja keinen fehler.

Reply:
was meinst du mit kann explizit übergeben? beispiel?

Reply:
Mir geht es darum, dass diese Zeile im FOR-Addon be_password dann keinen Fehler wirft: https://github.com/FriendsOfREDAXO/be_password/blob/abc1b1fbc28ba02cc382ab72ce31751127a25bdf/src/Controller/DefaultController.php#L108
Aber wenn wir den Parameter entfernen, gibt es ja keinen Fehler, auch wenn be_password die rules weiterhin explizit übergibt.

Reply:
Ok, d.h. Du willst das programatische überschreiben dieser optionen (nicht nur das property, sondern auch die 4 neuen rules) gar nicht zulassen, sondern nur via config.yml?